src/react/projects/spark-core-react/src/SprkHighlightBoard/SprkHighlightBoard.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import SprkHighlightBoard from './SprkHighlightBoard';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+it('should display a div element with the correct base class', () => {
+  const wrapper = shallow(<SprkHighlightBoard />);
+  expect(wrapper.find('div.sprk-c-HighlightBoard').length).toBe(1);
+});
+
+it('should not render an image if imgSrc is not provided', () => {
```

Comment:
I would like to make imgAlt required if imgSrc is provided. Think you could look into doing that?

Reply:
Custom validators coming in a later PR